docs/src/basic.md

```diff
@@ -88,6 +149,7 @@ Sometimes you may not have the data to insert into the table at creation time.
 In this case, you can create an empty table and specify the schema.
 
 === "Python"
+
```

Comment:
nit: should we also add js and rust code

Reply:
Oh i thought javascript / rust can not do it yet. I can double check

Reply:
also, should we add to our workflow to check the example rust code?

Reply:
javascript should be able to create an empty table

Reply:
Yea that'd be great. https://github.com/lancedb/lancedb/issues/875